frontend/src/components/VariantListPage/VariantListPage.tsx

```diff
@@ -179,8 +193,20 @@ const useVariantListAnnotation = (variantList: VariantList) => {
           variant_notes: Record<VariantId, string>;
           variant_calculations: VariantListCalculations;
           include_homozygotes_in_calculations: boolean;
+          taggedGroups: TaggedGroups;
         }) => {
           const selectedVariants = new Set(annotation.selected_variants);
+          const taggedGroups: {
+            A: { displayName: string; variantList: Set<VariantId> };
+            B: { displayName: string; variantList: Set<VariantId> };
+            C: { displayName: string; variantList: Set<VariantId> };
+            D: { displayName: string; variantList: Set<VariantId> };
+          } = {
+            A: { displayName: "Group A", variantList: new Set<VariantId>([]) },
+            B: { displayName: "Group B", variantList: new Set<VariantId>([]) },
+            C: { displayName: "Group C", variantList: new Set<VariantId>([]) },
+            D: { displayName: "Group D", variantList: new Set<VariantId>([]) },
```

Comment:
Here we'll need to use the information we've just gotten from the `get` request to populate the page with any information that comes back from the db on page load.

As it stands, we're saving information to the database, but if you refresh the page, all the saved information makes it to the frontend, but doesn't get displayed. The end result is that refreshing the page makes it look like you've lost all your tagged variants and naming of tags.